libs/execution_context/ebpf_maps.c

```diff
@@ -464,7 +464,8 @@ _find_array_map_entry(
     key_value = *(uint32_t*)key;
 
     if (key_value >= map->ebpf_map_definition.max_entries) {
-        return EBPF_INVALID_ARGUMENT;
+        *data = NULL;
```

Comment:
If this can pass null out, then line 457 has to change `_Outptr_` to `_Outptr_result_maybenull_`
This is why the analyze workflow is failing.